pom.xml

```diff
@@ -150,15 +150,16 @@
 					<version>3.2.0</version>
 				</plugin>
 
-				<plugin>
+					<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-checkstyle-plugin</artifactId>
-					<version>${checkstyle.plugin.version}</version>
```

Comment:
I deleted my `maven-checkstyle-plugin` from my local `.m2` and the Maven build was able to pull it back down without any problems. Are you using a Maven mirror or anything? I know it's a pain but have you tried building with a clean `.m2/repository`?